x-pack/legacy/plugins/siem/public/components/ml/conditional_links/entity_helpers.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const emptyEntity = (entity: string): boolean =>
+  entity.trim() === '' || (entity.startsWith('$') && entity.endsWith('$'));
+
+export const multipleEntities = (entity: string): boolean => entity.split(',').length > 1;
```

Comment:
nit: Consider is/has prefixes for `emptyEntity` and `multipleEntities` functions since they return George Booles.

Reply:
Thanks, I will do this in a follow up as this is time sensitive.